### DISCUSSION TITLE
Adding all Database Types to automation (webhook)

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#1628] [Help]
It would be great to have the possibility to use all data types of the database even in the automation. 

- Date/Time
- Bolean 
- ...

**Comment by Fabian1410** [comment]
Where is the feature :)

**Reply by mjashanks** [reply]
Hey @Fabian1410 -  We are working on it this Month. Will be available in the September release. At last! :)

**Reply by Fabian1410** [reply]
Nice to hear :)
